docs/Installation.md

```diff
@@ -72,9 +74,20 @@ Zonemaster::CLI, see the [declaration of prerequisites].
    ```
 
 
-### Installation on Debian or Ubuntu
+### Installation on Debian
 
-1) Update configuration of "locale"
+Using pre-built packages is the preferred method for Debian. If you prefer to
+install from CPAN instead, follow the steps for Ubuntu.
+
+1) Configure Zonemaster packages repository
```

Comment:
Is it correct to call this "Configure Zonemaster packages repository"? 
* "Add Zonemaster packages repository to repository list"
* "Configure OS packages repository list"

Reply:
"Add Zonemaster packages repository to repository list" is nice

Reply:
changed